src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Generated/Recipe.cs

```diff
@@ -0,0 +1,224 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using Amazon.CDK;
+using Amazon.CDK.AWS.ElasticBeanstalk;
+using Amazon.CDK.AWS.IAM;
+using Amazon.CDK.AWS.S3.Assets;
+using AWS.Deploy.Recipes.CDK.Common;
+
+using AspNetAppElasticBeanstalkLinux.Configurations;
+
+// This is a generated file from the original deployment recipe. It is recommended to not modify this file in order
+// to allow easy updates to the file when the original recipe that this project was created from has updates.
+// To customize the CDK constructs created in this file you should use the AppStack.CustomizeCDKProps() method.
+
+namespace AspNetAppElasticBeanstalkLinux
+{
+    using static AWS.Deploy.Recipes.CDK.Common.CDKRecipeCustomizer<Recipe>;
+
+    public class Recipe : Construct
+    {
+        private const string ENVIRONMENTTYPE_SINGLEINSTANCE = "SingleInstance";
+        private const string ENVIRONMENTTYPE_LOADBALANCED = "LoadBalanced";
+
+        public IRole? AppIAMRole { get; set; }
+
+        public IRole? BeanstalkServiceRole { get; set; }
+
+        public Asset? ApplicationAsset { get; set; }
+
+        public CfnInstanceProfile? Ec2InstanceProfile { get; set; }
+
+        public CfnApplicationVersion? ApplicationVersion { get; set; }
+
+        public CfnApplication? BeanstalkApplication { get; set; }
+
+        public CfnEnvironment? BeanstalkEnvironment { get; set; }
+
+        public Recipe(Construct scope, IRecipeProps<Configuration> props)
+            : base(scope, "Recipe")
```

Comment:
There should be a comment explaining why the construct id is constant here. And changing it will impact the stack.

Reply:
Done